### PR TITLE
[docs] Geo Failover

### PR DESCRIPTION
**Pull request by kaitlincarter-hc**
Updated the headers, code snippets, and some language.

Language updates include:
- "Failover Policy _Types_"
- cumbersome -> challenging
- api app -> banking- app
- templates -> prepared query template

### CODE REVIEW

File: website/source/docs/guides/geo-failover.html.md [hunk 1]
```diff
@@ -6,96 +6,115 @@ description: |-
...
 
 Failover is just another policy choice for a prepared query, it works in the same manner as the previous example and is similarly transparent to applications. The failover policy is configured using the `Failover` structure, which contains two fields, both of which are optional, and determine what happens if no healthy nodes are available in the local datacenter when the query is executed.
 
 - `NearestN` `(int: 0)` - Specifies that the query will be forwarded to up to `NearestN` other datacenters based on their estimated network round trip time using [network coordinates](/docs/internals/coordinates.html).
 
 - `Datacenters` `(array<string>: nil)` - Specifies a fixed list of remote datacenters to forward the query to if there are no healthy nodes in the local datacenter. Datacenters are queried in the order given in the list.
 
-The following sections show examples using these fields to implement different geo failover policies.
+The following examples use those fields to implement different geo failover policies methods.
```
[the remark this answers is not in the document]
Comment (kaitlincarter-hc) [reply website/source/docs/guides/geo-failover.html.md]: Yes it should! Good catch.

Comment (kaitlincarter-hc) [inline website/source/docs/guides/geo-failover.html.md]:
```suggestion
The following examples use those fields to implement different geo failover methods.
```

**Comment by kaitlincarter-hc** [comment]
@freddygv Thanks for the suggestion! I updated "banking-app" to "payments".

File: website/source/docs/guides/geo-failover.html.md [hunk 2]
```diff
@@ -6,96 +6,115 @@ description: |-
...
+    "Service": "payments",
     "Tags": ["v1.2.3"]
   }
 }' http://127.0.0.1:8500/v1/query
 
 {"ID":"fe3b8d40-0ee0-8783-6cc2-ab1aa9bb16c1"}
 ```
 
-This creates a prepared query called "api" that does a lookup for all instances of the "api" service with the tag "v1.2.3". This policy could be used to control which version of a "api" applications should be using in a centralized way. By [updating this prepared query](/api/query.html#update-prepared-query) to look for the tag "v1.2.4" applications could start to find the newer version of the service without having to reconfigure anything.
+This creates a prepared query called "banking-pp" that does a lookup for all instances of the "payments" service with the tag "v1.2.3". This policy could be used to control which version of a "payments" applications should be using in a centralized way. By [updating this prepared query](/api/query.html#update-prepared-query) to look for the tag "v1.2.4" applications could start to find the newer version of the service without having to reconfigure anything.
```
Comment (freddygv) [inline website/source/docs/guides/geo-failover.html.md]: One last edit:
```suggestion
This creates a prepared query called "payments" that does a lookup for all instances of the "payments" service with the tag "v1.2.3". This policy could be used to control which version of a "payments" applications should be using in a centralized way. By [updating this prepared query](/api/query.html#update-prepared-query) to look for the tag "v1.2.4" applications could start to find the newer version of the service without having to reconfigure anything.
```

**Comment by freddygv** [comment]
Great, almost there. Added one last suggestion inline.

**Pull request merged**